net/rs/client-gen/tests/snapshots/generator__basic_works.snap

```diff
@@ -0,0 +1,63 @@
+---
+source: client-gen/tests/generator.rs
+expression: "gen(idl, \"Basic\")"
+---
+using global::Sails.Remoting;
+using global::Sails.Remoting.Abstractions;
+using global::Sails.Remoting.Abstractions.Core;
+using global::System;
+using global::System.Collections.Generic;
+
+#pragma warning disable RCS0056 // A line is too long
```

Comment:
Why do we need this pragma, if generated code shouldn't be analyzed?

Reply:
I shouldn't, but pass